### ISSUE TITLE
Multiple calls to render_component

### ISSUE THREAD

**Issue by sealocal** [renchap/webpacker-react#8]
Thank you for taking initiative to develop `webpacker-react`!

I noticed that the view helper does not render iterations.

I'd like to render multiple components in my Rails views, for example:

```erb
<%= react_component('Hello', name: 'a component rendered from a view') %>

<% 10.times do %>
  <%= react_component('Hello', name: 'a component rendered from a view') %>
<% end %>

<% Tweet.all.each do |tweet| %>
  <%= react_component('Tweet', key: tweet.id, id: tweet.id, text: tweet.texts) %>
<% end %>

```

Is this intentional behavior, or perhaps a bug?

**Comment by renchap** [comment]
This is indeed a bug. I will have a look.

**Comment by sealocal** [comment]
OK, thanks for confirmation.  I can submit a PR with a fix tomorrow.

**Comment by renchap** [comment]
Fixed by #5
